make.config

```diff
@@ -45,6 +48,9 @@ ARPACK_NEEDS_LAPACK := 0
 # 2 -- level 1 checks plus extra runtime compiler checks
 DEBUG := 0
 
+# Nr. of processors used for testing
```

Comment:
Nr. of MPI processors used for testing (not aware of number of OMP shared memory threads)

Reply:
Done.